src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -197,6 +197,18 @@ def create(input_dataset, target, feature=None, validation_set='auto',
         raise TypeError("Unrecognized type for 'validation_set'."
             + validation_set_corrective_string)
 
+
+    original_dataset_size = len(dataset)
+    dataset = dataset.dropna(columns=feature)
+    if original_dataset_size != len(dataset):
+        print("Dropping " + str(original_dataset_size - len(dataset)) + " rows from the training dataset.")
```

Comment:
Could we indicate in the message why we're dropping the user's data?

Reply:
Dropping x rows for containing None values in the training dataset?